### PR TITLE
test: have typed playwright-core/* imports

### PR DESCRIPTION
**Pull request by mxschmitt**
We want to keep importing from lib/ internally, so tsconfig paths are not an option.

This adds a way for typed imports for our most used pw-core imports.

### CODE REVIEW

File: tests/global.d.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
+declare module '@playwright-core/utils' {
```
Comment (dgozman) [inline tests/global.d.ts:1]: Please add a thorough comment explaining why this file and what's put in here.

**Review by dgozman — APPROVED** [review]
This looks good. However, we use a different technique to export typed values from bundles, and also have an `internalsForTest` file that is along the same lines. It would be great to have a single way of doing things.

**Comment by pavelfeldman** [comment]
What's happening here?

**Comment by dgozman** [comment]
Closing as per offline discussion. This add extra complexity and does not solve enough problems. For example, "find all usages" does not really work unless you open the test.

**Pull request closed without merging**